CHANGELOG.rst

```diff
@@ -6,26 +6,22 @@ Change Log
    in this file.  It adheres to the structure of http://keepachangelog.com/ ,
    but in reStructuredText instead of Markdown (for ease of incorporation into
    Sphinx documentation and the PyPI description).
-   
+
    This project adheres to Semantic Versioning (http://semver.org/).
 
 .. There should always be an "Unreleased" section for changes pending release.
 
-2020-07-08
-~~~~~~~~~~
-* Modified scripts/feature_toggle_report_generator to work based on envs
+[0.3.0] - 2020-07-22
+~~~~~~~~~~~~~~~~~~~~
 
-2020-06-30
-~~~~~~~~~~
-* Added CsvResnderer
+* Add ``featuretoggles`` Sphinx extension
+* Modified scripts/feature_toggle_report_generator to work based on envs
```

Comment:
FYI: @jinder1s will soon be merging https://github.com/edx/edx-toggles/pull/40 as well.  Not sure whether or not will will need to those entries as well, or if he'll move this to 0.4.0.